### DISCUSSION TITLE
Very serious errors when working with user groups in version 0.4.0 beta 3.

### DISCUSSION THREAD

**Discussion by AlexAlex88888** [xchacha20-poly1305/husi#21] [Q&A]
When deleting, adding, deleting configurations after tests, dynamically changing the number of configurations in user groups does not work at all. Moreover, the groups begin to be chaotically duplicated and are displayed more or less correctly only after restarting the program. Displaying the number of configurations after tests and subsequent removal of non-working ones, as well as after importing a group, does not work even in the default group. 
Sincerely.

**Comment by xchacha20-poly1305** [comment]
Because of too many bugs, so this function will be removed.
